.github/workflows/scripts/homebrew.sh

```diff
@@ -52,7 +63,11 @@ function install_arm64() {
 
 function install_intel() {
     export CPU="x86_64"
+    export HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK="true"
```

Comment:
Used in the arm64 case already, avoids wasting time on updating unrelated packages.